{"cells":[{"source":"<a href=\"https://www.kaggle.com/code/douglashsm/llm-crutch?scriptVersionId=263213132\" target=\"_blank\"><img align=\"left\" alt=\"Kaggle\" title=\"Open in Kaggle\" src=\"https://kaggle.com/static/images/open-in-kaggle.svg\"></a>","metadata":{},"cell_type":"markdown"},{"cell_type":"code","execution_count":1,"id":"9c7c697a","metadata":{"execution":{"iopub.execute_input":"2025-09-21T20:33:40.96182Z","iopub.status.busy":"2025-09-21T20:33:40.961393Z","iopub.status.idle":"2025-09-21T20:37:03.718952Z","shell.execute_reply":"2025-09-21T20:37:03.71755Z"},"papermill":{"duration":202.762761,"end_time":"2025-09-21T20:37:03.720899","exception":false,"start_time":"2025-09-21T20:33:40.958138","status":"completed"},"tags":[]},"outputs":[{"name":"stdout","output_type":"stream","text":["\u001b[33mWARNING: Retrying (Retry(total=4, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<pip._vendor.urllib3.connection.HTTPSConnection object at 0x785f89c35c50>: Failed to establish a new connection: [Errno -3] Temporary failure in name resolution')': /simple/streamlit/\u001b[0m\u001b[33m\r\n","\u001b[0m\u001b[33mWARNING: Retrying (Retry(total=3, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<pip._vendor.urllib3.connection.HTTPSConnection object at 0x785f89b02d90>: Failed to establish a new connection: [Errno -3] Temporary failure in name resolution')': /simple/streamlit/\u001b[0m\u001b[33m\r\n","\u001b[0m\u001b[33mWARNING: Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<pip._vendor.urllib3.connection.HTTPSConnection object at 0x785f89af8b90>: Failed to establish a new connection: [Errno -3] Temporary failure in name resolution')': /simple/streamlit/\u001b[0m\u001b[33m\r\n","\u001b[0m\u001b[33mWARNING: Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<pip._vendor.urllib3.connection.HTTPSConnection object at 0x785f89b18110>: Failed to establish a new connection: [Errno -3] Temporary failure in name resolution')': /simple/streamlit/\u001b[0m\u001b[33m\r\n","\u001b[0m\u001b[33mWARNING: Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<pip._vendor.urllib3.connection.HTTPSConnection object at 0x785f89b21f50>: Failed to establish a new connection: [Errno -3] Temporary failure in name resolution')': /simple/streamlit/\u001b[0m\u001b[33m\r\n","\u001b[0m\u001b[31mERROR: Could not find a version that satisfies the requirement streamlit (from versions: none)\u001b[0m\u001b[31m\r\n","\u001b[0m\u001b[31mERROR: No matching distribution found for streamlit\u001b[0m\u001b[31m\r\n","\u001b[0mNote: you may need to restart the kernel to use updated packages.\n"]}],"source":["# Instala todas as bibliotecas do requirements.txt\n","%pip install streamlit google-generativeai google-cloud-bigquery pandas db-dtypes plotly"]}],"metadata":{"kaggle":{"accelerator":"none","dataSources":[],"dockerImageVersionId":31089,"isGpuEnabled":false,"isInternetEnabled":false,"language":"python","sourceType":"notebook"},"kernelspec":{"display_name":"Python 3","language":"python","name":"python3"},"language_info":{"codemirror_mode":{"name":"ipython","version":3},"file_extension":".py","mimetype":"text/x-python","name":"python","nbconvert_exporter":"python","pygments_lexer":"ipython3","version":"3.11.13"},"papermill":{"default_parameters":{},"duration":209.063625,"end_time":"2025-09-21T20:37:04.244512","environment_variables":{},"exception":null,"input_path":"__notebook__.ipynb","output_path":"__notebook__.ipynb","parameters":{},"start_time":"2025-09-21T20:33:35.180887","version":"2.6.0"}},"nbformat":4,"nbformat_minor":5}